synapse/config/workers.py

```diff
@@ -28,7 +28,7 @@ def read_config(self, config):
         if self.worker_app == "synapse.app.homeserver":
             self.worker_app = None
 
-        self.worker_listeners = config.get("worker_listeners")
+        self.worker_listeners = config.get("worker_listeners", [])
```

Comment:
this looks like it will fix https://github.com/matrix-org/synapse/issues/4783 ?

Reply:
Ooh, yes!